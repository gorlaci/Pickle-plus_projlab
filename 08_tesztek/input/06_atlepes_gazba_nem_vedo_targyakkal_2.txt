create room r1
create room r2
setroom r2 gas true
create student s
create teacher t
create beerglass b
create falsemask fm

neighbour r1 r2
addperson s r1
addperson t r2
add b s
add fm s
enter r2 s

status s
status