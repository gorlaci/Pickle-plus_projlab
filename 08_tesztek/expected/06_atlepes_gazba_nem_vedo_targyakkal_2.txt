
s
location: r2
itemsInHand: 
stunRemaining: 3
Game over
